src/bowlingCl.js

```diff
@@ -0,0 +1,125 @@
+function BowlingCl () {
+
+this._currentFrameScores = []
+this._runningTotal = []
+this._previousFrame = []
+this.STRIKE_COUNT = 0
+this.FRAME_COUNT = 0
+
+};
+
+
+BowlingCl.prototype.inputFrame = function(roll1, roll2) {
+
+  var string = roll1;
+  var rollnumber1 = string*1;
+  var string2 = roll2;
+  var rollnumber2 = string2*1;
+
```

Comment:
Could just be...
  var rollnumber1 = roll1 * 1;
  var rollnumber2 = roll2 * 1;